subcommittees/mentoring/README.md

```diff
@@ -1,70 +1,86 @@
 # The Mentoring Subcommittee
 
-## Purpose and Scope
+## Our mission
+
 The Mentoring Subcommittee was established in 2015 to develop and maintain a mentorship program to support instructors as they progress through training, teaching, curriculum development, and other community-related activities. We help promote community-building and networking by providing (virtual) spaces where instructors from all over the world can share teaching success stories and discuss strategies for overcoming challenges. 
 
-## What We Do 
+## What We Do
 
 ### Instructor Discussion Sessions
-Each week, we host four instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
+
+Each week, volunteers host two instructor discussion sessions. The sessions provide a forum where experienced and novice instructors can discuss past and/or upcoming workshops. Instructors who have just taught, are about to teach, or have just completed instructor training will  receive an email invitation to these meetings, but these sessions are open to anyone. 
 
 To find an upcoming session, visit this link: http://pad.software-carpentry.org/instructor-discussion
 
-### Instructor-Helper Retreat
-In November 2015, Software Carpentry and Data Carpentry instructors and helpers got together together at sites around the world for a day of sharing skills, trying out new lessons and ideas and discussing all things instructor and helper related. 
+### Mentoring Program
 
-To see the schedule for the 2015 retreat, visit: http://swcarpentry.github.io/instructor-retreat-2015/
+Need to add
 
-### Archived: Pre- and Post- Workshop Debriefing Session
+### Coming Soon: Regional Communities Sub-Committee
```

Comment:
This is #158.  I'd leave this off until then, and once #158 lands add a reference in a [Related Projects][1] section.

[1]: https://github.com/swcarpentry/board/tree/45f7e9606e58c85cc9fbf3f534416b39fd74a7a2/subcommittees/template#related-projects